app/controllers/users_controller.rb

```diff
@@ -16,6 +16,11 @@ def create
     end
   end
 
+  def show
+    user = User.find(params[:id])
+    @recommendables = user.recommendables
```

Comment:
What about renaming the join table like `has_many :recommendations, through: recommended_items`? I know `items` is vague.

Not that this helps, but here was what I did in Consoom (https://github.com/adarsh/consoom/blob/master/app/models/recommendation.rb#L3-L4)